### PR TITLE
Update sbt + scala + CI docker image

### PR DESCRIPTION
**Pull request by brandon-leapyear**
Got errors updating the monorepo to use sbt 1.7

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -21,7 +21,7 @@ repos:
       - id: check-vcs-permalinks
       - id: detect-private-key
   - repo: https://github.com/coyainsurance/pre-commit-scalafmt
-    rev: 1.0.1
+    rev: 1163a9a9c869a63e74d05937ca61c6e1bb364b87
```
Comment (liam-ly) [inline .pre-commit-config.yaml:24]: what was wrong with it?

File: .circleci/config.yml [hunk 2]
```diff
@@ -8,7 +8,10 @@ executors:
         resource_class: small
     sbt:
         docker:
-            - image: leapyear1/sbt:8u265-jdk-slim-buster_1.3.13
+            - auth:
+                password: $ARTIFACTORY_PASSWORD
+                username: $ARTIFACTORY_USER
+              image: artifacts.leapyear.io/dockerhub-remote/sbtscala/scala-sbt:8u332_1.3.13_2.12.10
```
Comment (liam-ly) [inline .circleci/config.yml]: 😮 oh good finally

File: src/main/scala/dasho/DashOPlugin.scala [hunk 3]
```diff
@@ -61,10 +61,10 @@ object DashOPlugin extends AutoPlugin {
       dashOHome.value.getOrElse(throw new MessageOnlyException("dashOHome is not set"))
 
     // Get classPaths for DashO to inspect
-    val classPaths: Seq[File] = (Keys.fullClasspath in Runtime).value.files
+    val classPaths: Seq[File] = (Runtime / Keys.fullClasspath).value.files
```
Comment (liam-ly) [inline src/main/scala/dasho/DashOPlugin.scala:64]: I assume this is just a deprecation replacement?

File: .circleci/@orbs.yml [hunk 4]
```diff
@@ -1,2 +1,3 @@
 orbs:
+  aws-cli: circleci/aws-cli@2.0.2
```
Comment (liam-ly) [inline .circleci/@orbs.yml:2]: "Fix CI" what was wrong before?

Comment (liam-ly) [reply .circleci/@orbs.yml:2]: ah - you replaced the image and the new one doesn't have awscli ✅

**Review by liam-ly — APPROVED** [review]
I'm missing some context, but LGTM

File: .pre-commit-config.yaml (hunk 1 above)
Comment (brandon-leapyear) [reply .pre-commit-config.yaml:24]: `pre-commit-scalafmt` runs a script that downloads another script; the link to the second script was broken and they updated it: https://github.com/coyainsurance/pre-commit-scalafmt/commit/b10049d054858e0f883973a419365135a7c2b078

They haven't released a new version yet :(

File: src/main/scala/dasho/DashOPlugin.scala (hunk 3 above)
Comment (brandon-leapyear) [reply src/main/scala/dasho/DashOPlugin.scala:64]: Yup; deprecated in 1.5

**Pull request merged**